### DISCUSSION TITLE
posts/HelloWorld/

### DISCUSSION THREAD

**Discussion by giscus** [HalfSweet/Blog#18] [comment]
# posts/HelloWorld/

今天搭建好了属于自己的一个博客站，希望今后能保持记录的习惯，写下自己遇到的坑以及收获。这是博客的一个启程，也是自己的一个新起点。 Hello, HalfSweet!

https://halfsweet.cn/posts/HelloWorld/

**Comment by HalfSweet** [comment]
评论测试
